source/fundamentals/builders/sort.txt

```diff
@@ -71,86 +58,91 @@ sample collection, ``sort_example``, that contains the following documents:
    {"_id": 2, "letter": "a", "food": "donuts and coffee"}
    {"_id": 6, "letter": "c", "food": "maple donut"}
 
+The Sorts Class
+---------------
 
-Sorting Direction
------------------
+The ``Sorts`` class is a builder that provides static factory methods for all sort
+criteria operators supported by MongoDB. These methods return a ``Bson`` object
+that you can pass to the  ``sort()`` method of a ``FindIterable`` instance or to 
+``Aggregates.sort()``. If you want to learn more about ``Aggregates``, see our 
+:doc:`guide on the Aggregates builder </fundamentals/builders/aggregates>`.
 
-The ``Sorts`` class provides methods for specifying the direction of your sort.
-The direction of your sort can either be **ascending** or **descending**.
-An ascending sort orders your results from smallest to largest. A
-descending sort orders your results from largest to smallest.
+The following two code snippets are equivalent, however we recommend you use the
```

Comment:
Avoid joining two sentences with a comma:
```suggestion
The following two code snippets are equivalent. However, we recommend you use the
```

Reply:
Thank you for the suggestion. I am not sure I agree with this edit, as using a period makes me read it as two separate thoughts (where in my head it is one thought "code snippets do the same thing but one is preferred"). Will commit suggestion, but would appreciate follow up discussion.